src/Message/CacheHeaderMessage.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+declare(strict_types=1);
+
+namespace ADS\Bundle\ApiPlatformEventEngineBundle\Message;
+
+use EventEngine\Schema\TypeSchema;
+
+interface CacheHeaderMessage
+{
+    /** @return array<mixed> */
+    public static function __cacheHeaders(): array;
```

Comment:
Can you split things up here __maxAge, __sharedMaxAge, ... and build the array in the ResourceMetadataCollectionFactory?